pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -42,6 +42,10 @@ buildPythonPackage rec {
         ++ lib.optional phantomjsSupport phantomjs2;
     in [ ''--prefix PATH : "${lib.makeBinPath packagesToBinPath}"'' ];
 
+  preBuild = ''
```

Comment:
Can't we do
```
setupPyBuildFlags = [
  "build_lazy_extractors"
];
```

Reply:
Yes, thanks! It looks like that works, so PR updated.